### DISCUSSION TITLE
What is this 'use client' usage?

### DISCUSSION THREAD

**Discussion by michaelgruner** [vercel/next.js#67500] [Help]
### Summary

I think I understand client/server components and server actions. Then I stumbled across this:

https://mui.com/material-ui/integrations/nextjs/#theming

src/theme.js
```
'use client';
import { Roboto } from 'next/font/google';
import { createTheme } from '@mui/material/styles';

const roboto = Roboto({
  weight: ['300', '400', '500', '700'],
  subsets: ['latin'],
  display: 'swap',
});

const theme = createTheme({
  typography: {
    fontFamily: roboto.style.fontFamily,
  },
});

export default theme;
```

What is that? It is  not a component but it is declared as 'use client' and can be imported and used from a server component. 

What is going on here? Is this documented?

**Comment by romhenri** [comment]
Hello, man. About be component, in a way, it can be considered a component, as part of the application.

Normal to be called by a server-component.

It would be strange if it were the other way around, a client calling a server, which would be an inversion.

In this case, I believe it is a client due to the way Material UI works and the font is loaded on the client.
